automation/src/test/scala/org/broadinstitute/dsde/firecloud/config/Config.scala

```diff
@@ -25,38 +48,35 @@ object Config {
   }
 
   object Users {
-    val notSoSecretPassword = users.getString("notSoSecretPassword")
-
-    val dumbledore = Credentials(users.getString("dumbledore"), notSoSecretPassword)
-    val voldemort = Credentials(users.getString("voldemort"), notSoSecretPassword)
-    val admin = dumbledore
+    private val notSoSecretPassword = users.getString("notSoSecretPassword")
+    private val userDataJson = JSON.parseFull(scala.io.Source.fromFile(users.getString("userDataPath")).getLines.mkString).get.asInstanceOf[Map[String, Map[String,String]]]
 
-    val hermione = Credentials(users.getString("hermione"), notSoSecretPassword)
-    val owner = hermione
+    def makeCredsMap(jsonMap: Map[String, String]): Map[String, Credentials] = {
+      for((k,v) <- jsonMap) yield (k, Credentials(v, notSoSecretPassword))
+    }
 
-    val mcgonagall = Credentials(users.getString("mcgonagall"), notSoSecretPassword)
-    val snape = Credentials(users.getString("snape"), notSoSecretPassword)
-    val curator = mcgonagall
+    val Admins = UserSet(makeCredsMap(userDataJson("admins")))
+    val Owners = UserSet(makeCredsMap(userDataJson("owners")))
+    val Curators = UserSet(makeCredsMap(userDataJson("curators")))
+    val Temps = UserSet(makeCredsMap(userDataJson("temps")))
+    val AuthDomainUsers = UserSet(makeCredsMap(userDataJson("authdomains")))
+    val Students = UserSet(makeCredsMap(userDataJson("students")))
 
-    val harry = Credentials(users.getString("harry"), notSoSecretPassword)
-    val ron = Credentials(users.getString("ron"), notSoSecretPassword)
-    val draco = Credentials(users.getString("draco"), notSoSecretPassword)
+    // defaults
```

Comment:
there were a few places where i opted to use hermione vs a user-pool chosen user, since she has some additional configuration around her regarding some of the methods/method configs we build-in during fiab populate.

Reply:
If that's fixed now in the user setup, can we avoid choosing a specific user? I worry that making these available will tempt others to use them when they should be using the pool.

Reply:
And if we really need to use specific users for specific reasons, let's rename the val to be explicit about who we're using and add some comments here about why the users are special (and remove any that we don't need).